src/test/java/unicash/ui/StyleSheetTest.java

```diff
@@ -0,0 +1,154 @@
+package unicash.ui;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static unicash.ui.StyleSheet.BLUE_OFFSET;
+import static unicash.ui.StyleSheet.BRIGHTNESS_OFFSET;
+import static unicash.ui.StyleSheet.BRIGHTNESS_THRESHOLD;
+import static unicash.ui.StyleSheet.COLOR_END_INDEX;
+import static unicash.ui.StyleSheet.COLOR_START_INDEX;
+import static unicash.ui.StyleSheet.DEFAULT_BACKGROUND_COLOR_HEX;
+import static unicash.ui.StyleSheet.FONT_STYLE_BOLD;
+import static unicash.ui.StyleSheet.GREEN_OFFSET;
+import static unicash.ui.StyleSheet.IS_YELLOW_SKEW;
+import static unicash.ui.StyleSheet.MAX_COLOUR_VALUE;
+import static unicash.ui.StyleSheet.RED_OFFSET;
+import static unicash.ui.StyleSheet.TEXT_FILL_RED;
+import static unicash.ui.StyleSheet.USE_DEFAULT_STYLE;
+import static unicash.ui.StyleSheet.absoluteColorAddition;
+import static unicash.ui.StyleSheet.adjustBrightness;
+import static unicash.ui.StyleSheet.getColorOutput;
+import static unicash.ui.StyleSheet.padHexString;
+
+import org.junit.jupiter.api.Test;
+
+public class StyleSheetTest {
+
+    @Test
+    public void testConstants() {
+        assertEquals("-fx-text-fill: red", TEXT_FILL_RED);
+        assertEquals("-fx-font-weight: bold", FONT_STYLE_BOLD);
+    }
+
+    @Test
+    public void hexValues_nonNegativeValue_returnTrue() {
+        assertTrue(COLOR_START_INDEX >= 0);
+        assertTrue(COLOR_END_INDEX <= 6);
+
+    }
+
+    @Test
+    public void offsetValues_withinRange_returnTrue() {
+        assertTrue(BRIGHTNESS_THRESHOLD >= 0);
+        assertTrue(RED_OFFSET >= -255 && RED_OFFSET <= 255);
+        assertTrue(GREEN_OFFSET >= -255 && GREEN_OFFSET <= 255);
+        assertTrue(BLUE_OFFSET >= -255 && BLUE_OFFSET <= 255);
+
+    }
+
+    @Test
+    public void combinedColorBrightness_lessThanThreshold_returnTrue() {
+        String color = StyleSheet.getBrightCategoryColorFromHash(new Object());
+        int r = Integer.parseInt(color.substring(1, 3), 16);
+        int g = Integer.parseInt(color.substring(3, 5), 16);
+        int b = Integer.parseInt(color.substring(5, 7), 16);
+
+        int totalBrightness = r + g + b;
+
+        assertTrue(totalBrightness >= BRIGHTNESS_THRESHOLD);
+    }
+
+    @Test
+    public void combinedColorBrightness_isColorSkewed_returnTrue() {
+        String color = StyleSheet.getBrightCategoryColorFromHash(new Object() {
+            @Override
+            public int hashCode() {
+                return 11111111;
+            }
+        });
```

Comment:
If the reason for making the input an `Object` for testing purposes, I strongly believe constraining the inputs to `Category` or `String` and then properly generating those values to generate the expected hex codes should be more appropriate